src/api/api.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import { LOCAL_SERVER, API_ADDRESS, QUERY_TYPE } from './constants';
+
+function getLocalData() {
```

Comment:
for what using custom hooks? You can use just JS service for this case, without React